config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1, defaults: {format: 'json'} do
+      resources :categories, only: [:index, :show]
+      resources :properties, only: %i[index show create update destroy]
+      resources :reservation_criterias, only: [:index, :show, :create, :update, :destroy]
```

Comment:
- I think reservation_criterias routes is not necessary, because we only displaying in property routes.